Source/Core/Core/CoreTiming.cpp

```diff
@@ -280,7 +281,8 @@ static void AddEventToQueue(Event* ne)
 // than Advance
 void ScheduleEvent(int cyclesIntoFuture, int event_type, u64 userdata)
 {
-	_assert_msg_(POWERPC, Core::IsCPUThread(), "ScheduleEvent from wrong thread");
+	// TODO: Fix UI thread safety problems, and enable this assertion
+	//_assert_msg_(POWERPC, Core::IsCPUThread(), "ScheduleEvent from wrong thread");
```

Comment:
This assert tells us every use of GetUsbPointer() in DolphinWX/FrameTools.cpp is buggy.  I'm not familiar enough with how the UI code to fix that properly without spending a lot of time studying it.  I could change CWII_IPC_HLE_Device_usb_oh1_57e_305::EnqueueReply to use the threadsafe codepath, but that doesn't actually make the Wiimote handling code threadsafe, and it affects a bunch of other stuff.